task/jib-maven/0.1/jib-maven.yaml

```diff
@@ -55,6 +55,8 @@ spec:
     - name: $(params.CACHE)
       mountPath: /workspace/.cache
       subPath: jib-cache
+    securityContext:
+      privileged: true
```

Comment:
I guess we don't need changes in this version as we are mounting certs in version 0.3